src/codeu/chat/server/Server.java

```diff
@@ -66,14 +73,44 @@
   private final Relay relay;
   private Uuid lastSeen = Uuid.NULL;
 
-  private static ServerInfo info;
 
-  public Server(final Uuid id, final Secret secret, final Relay relay) {
+  public Server(final Uuid id, final Secret secret, final Relay relay) throws IOException {
```

Comment:
Hi Alan, I am confused about handling the exception. I thought when we put the code in the try block and catching the exception is when we are handling it.